airbyte-integrations/connectors/destination-astra/pyproject.toml

```diff
@@ -18,7 +18,7 @@ include = "destination_astra"
 
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
-airbyte-cdk = {version = "0.81.6", extras = ["vector-db-based"]}
+airbyte-cdk = {version = "0.1.4", extras = ["vector-db-based"]}
```

Comment:
@alafanechere wow wow wow there is a bug in `up_to_date` that sets `version=` everywhere, even if it's not in the beginning of the string. Be very careful about this, needs to be fixed before you merge.